src/ORM/ValidationResult.php

```diff
@@ -236,6 +236,7 @@ public function __unserialize(array $data): void
      */
     public function serialize()
     {
+        Deprecation::notice('4.12.0', 'Will be removed in 5.0');
```

Comment:
Add the version to the phpdoc, and replace the message with something standardised

Reply:
Updated